### DISCUSSION TITLE
Trying to run some example file but nothing show up

### DISCUSSION THREAD

**Discussion by Waynechuuuu** [marcomusy/vedo#1097] [Q&A] [answered]
i can run the file correctly when i don't need to output 3D object but only use the console.
![image](https://github.com/marcomusy/vedo/assets/138635577/9c0d41c3-c2eb-4c7d-b62b-fbce68939bf8)
like this case it works well
![image](https://github.com/marcomusy/vedo/assets/138635577/3f0b05f5-55e2-4c87-9b22-c4af4eb8129d)
but when i trying to output some image or 3D object, it didn't work and nothing show up

**Comment by marcomusy** [comment, marked as answer]
Is that spyder?
you can try
```
pip install -U git+https://github.com/marcomusy/vedo.git
```

to have a independent window to pop up you can set it trough
```
vedo.settings.default_backend = "vtk"
```

Eg
```py
import vedo
vedo.settings.default_backend = "vtk"
plt = vedo.Sphere().show(axes=1)
plt.close()
```

**Reply by Waynechuuuu** [reply]
THX very much！ It works :)
